ops/docker/Dockerfile.packages

```diff
@@ -81,6 +81,8 @@ COPY .git/ ./.git
 COPY .gitmodules ./.gitmodules
 RUN git submodule update --init --recursive
 
+COPY --from=1password/op:2 /usr/local/bin/op /usr/local/bin/op
```

Comment:
Does this need to go into every single build target? Or can it just go into a subset of them?

Since chain-mon is a parent package for a bunch of different entrypoints, we've historically added a new build target to it and added and customizations there.

Reply:
We currently only build `chainmon` and use different entry points for each service. If we would produce separate builds, we would multiply space used to store those images.

Reply:
Ok, let's do this:

- Let's add this here for now. Then, let's address the issues preventing us from building multisig-mon as a standalone package.